src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(x) {
+    if (x || x === 0) {
```

Comment:
```suggestion
    if (x || x === 0) {
```
можна поміняти перевірки місцями і перевіряти
```
 if (x === undefined) {
```